src/components/form-helpers/form-helpers.js

```diff
@@ -526,6 +526,27 @@ export const CheckboxFieldItem = ({
   </FieldItem>
 );
 
+export const NumberField = ({
+  className,
+  fieldName,
+  placeholder,
+  required,
+  id,
+  onChange,
+  ...rest
+}) => (
+  <Field
+    type="number"
+    className={className}
+    name={fieldName}
+    id={id}
+    onChange={onChange}
+    placeholder={placeholder}
```

Comment:
These 4 fields could be part of the rest, I mean, these lines are redundant, right? or I am missing something?

```diff

export const NumberField = ({
-  className,
  fieldName,
-  placeholder,
   required,
-  id,
-  onChange,
   ...rest
 }) => (

   <Field
     type="number"
-    className={className}
     name={fieldName}
-    id={id}
-    onChange={onChange}
-    placeholder={placeholder}
     validate={createRequiredValidation(required)}
     {...rest}
```

You could even rename `fieldName` as `name` and it will be also redundant.